manage_arkime/core/user_config.py

```diff
@@ -13,6 +15,30 @@ class UserConfig:
     pcapDays: int
     viewerPrefixList: str = None
 
+    def __init__(self, expectedTraffic: float, spiDays: int, historyDays: int, replicas: int, pcapDays: int, viewerPrefixList: str = None):
+        if (expectedTraffic is None):
+            expectedTraffic = MINIMUM_TRAFFIC
+
+        if (spiDays is None):
+            spiDays = DEFAULT_SPI_DAYS
+
+        if (historyDays is None):
+            historyDays = DEFAULT_HISTORY_DAYS
+
+        if (replicas is None):
+            replicas = DEFAULT_REPLICAS
+
+        if (pcapDays is None):
+            pcapDays = DEFAULT_S3_STORAGE_DAYS
```

Comment:
Recommend not changing the values of the arguments themselves; Python is pass-by-reference so this should change the value of the variable in the outside scope that you're invoking the constructor from.  A better way would be to just set the `self.*` members based on the conditional instead.  That will prevent "surprises" from the caller's perspective.